src/actions/base.ts

```diff
@@ -124,9 +124,9 @@ export class BaseAction {
       return false;
     }
     if (
+      this.mustBeFirstKey &&
       vimState.recordedState.getCurrentCommandWithoutCountPrefix().length - keysPressed.length >
```

Comment:
`this.mustBeFirstKey` is false for all but 3 actions, makes sense to take advantage of short-circuit evaluation here to avoid work.